doc/tutorials/nvmx_virtualization.rst

```diff
@@ -101,39 +101,47 @@ with these settings:
 
 #. Configure system level features:
 
-   - Select ``y`` on :option:`hv.FEATURES.NVMX_ENABLED` to enable nested virtualization
+   - Edit :option:`hv.FEATURES.NVMX_ENABLED` to `y` to enable nested virtualization
 
-   - Select ``SCHED_NOOP`` on :option:`hv.FEATURES.SCHEDULER`
+   - Edit :option:`hv.FEATURES.SCHEDULER` to ``SCHED_NOOP`` to disable CPU sharing
 
-     .. figure:: images/nvmx_cfg_1.png
-        :width: 400px
-        :align: center
-
-        Setting NVMX_ENABLED and SCHEDULER with configuration tool
+     .. code-block:: xml
+        :emphasize-lines: 3,18
+
+        <FEATURES>
+            <RELOC>y</RELOC>
+            <SCHEDULER>SCHED_NOOP</SCHEDULER>
+            <MULTIBOOT2>y</MULTIBOOT2>
+            <ENFORCE_TURNOFF_AC>y</ENFORCE_TURNOFF_AC>
+            <RDT>
+                <RDT_ENABLED>n</RDT_ENABLED>
+                <CDP_ENABLED>y</CDP_ENABLED>
+                <CLOS_MASK>0xfff</CLOS_MASK>
+                <CLOS_MASK>0xfff</CLOS_MASK>
+                <CLOS_MASK>0xfff</CLOS_MASK>
+                <CLOS_MASK>0xfff</CLOS_MASK>
+                <CLOS_MASK>0xfff</CLOS_MASK>
+                <CLOS_MASK>0xfff</CLOS_MASK>
+                <CLOS_MASK>0xfff</CLOS_MASK>
+                <CLOS_MASK>0xfff</CLOS_MASK>
+            </RDT>
+            <NVMX_ENABLED>y</NVMX_ENABLED>
+            <HYPERV_ENABLED>y</HYPERV_ENABLED>
 
 #. In each guest VM configuration:
 
-   - Select ``GUEST_FLAG_NVMX_ENABLED`` on :option:`vm.guest_flags.guest_flag` on the SOS VM section
+   - Edit :option:`vm.guest_flags.guest_flag` on the SOS VM section and add ``GUEST_FLAG_NVMX_ENABLED``
      to enable the nested virtualization feature on the Service VM.
-   - Select ``GUEST_FLAG_LAPIC_PASSTHROUGH`` on :option:`vm.guest_flags.guest_flag` to enable local
+   - Edit :option:`vm.guest_flags.guest_flag` and add ``GUEST_FLAG_LAPIC_PASSTHROUGH`` to enable local
      APIC passthrough on the Service VM.
-
-     .. figure:: images/nvmx_cfg_3.png
-        :width: 700px
-        :align: center
-
-        Service VM (SOS) ``guest_flag`` settings
-
    - Edit :option:`vm.cpu_affinity.pcpu_id` to assign ``pCPU`` IDs to run the Service VM. If you are
      using debug build and need the hypervisor console, don't assign
      ``pCPU0`` to the Service VM.
 
-     You may need to manually edit the ACRN scenario XML configuration file to edit the ``pcpu_id`` for the Service VM (SOS):
-
      .. code-block:: xml
-        :emphasize-lines: 5,6,7
+        :emphasize-lines: 5,6,7,10,11
```

Comment:
Let's put the note back in above this code block saying:
```
.. note:: Normally you'd use the configuration tool GUI to edit the scenario XML file.
   The tool wasn't updated in time for the v2.5 release, so you'll need to manually edit
   the ACRN scenario XML configuration file to edit the ``pcpu_id`` , ``guest_flags``,
   ``legacy_vuart``, and ``console_vuart`` settings for the Service VM (SOS), as shown below:
```

Reply:
Thanks.